package.json

```diff
@@ -6,23 +6,37 @@
   "scripts": {
     "watch": "webpack --watch --config webpack.dev.js",
     "build": "webpack --config webpack.prod.js",
-    "watch-poll": "webpack --watch --watch-poll --config webpack.dev.js"
+    "watch-poll": "webpack --watch --watch-poll --config webpack.dev.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
   },
   "author": "Event Espresso",
   "license": "GPL-2.0",
   "browserslist": [
     "extends @wordpress/browserslist-config"
   ],
+  "engines": {
+    "node": ">=8.0.0",
+    "npm": ">=5.0.0"
+  },
   "devDependencies": {
     "@wordpress/babel-preset-default": "^1.1.0",
     "@wordpress/browserslist-config": "^2.1.2",
     "assets-webpack-plugin": "^3.5.1",
     "autoprefixer": "^8.2.0",
     "babel-core": "^6.26.0",
+    "babel-eslint": "^8.2.2",
     "babel-loader": "^7.1.2",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-polyfill": "^6.26.0",
     "clean-webpack-plugin": "^0.1.18",
     "css-loader": "^0.28.11",
+    "eslint": "^4.19.1",
+    "eslint-config-wordpress": "^2.0.0",
+    "eslint-plugin-jest": "^21.15.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-react": "^7.7.0",
+    "eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#1774343f6226052a46b081e01db3fca8793cc9f1",
```

Comment:
Changes in these files DO mean you'll need to run npm install when they get propagated to upstream branches.